common/util.go

```diff
@@ -128,9 +130,39 @@ func BytesToVideoProfile(txData []byte) ([]ffmpeg.VideoProfile, error) {
 	return profiles, nil
 }
 
+func DefaultProfileName(width int, height int, bitrate int) string {
+	return fmt.Sprintf("%dx%d_%d", width, height, bitrate)
+}
+
+func FFmpegProfiletoNetProfile(ffmpegProfiles []ffmpeg.VideoProfile) ([]*net.VideoProfile, error) {
+	profiles := []*net.VideoProfile{}
```

Comment:
will not make much difference here, but still it's a good practice to do
`profiles := make([]*net.VideoProfile, 0, len(ffmpegProfiles))`